### ISSUE TITLE
adjustable column widths for source stream names

### ISSUE THREAD

**Issue by jrhizor** [airbytehq/airbyte#9735]
It's very difficult for users to understand stream names (for example in the Facebook source) due to long stream names and prefixes that obfuscate the differences.

Ideally the user could stretch the prefix section out, or we could make sure that the whole stream names are shown at all times.

Blocked on design for this.

**Comment by jrhizor** [comment]
@Jamakase is this something easy to do or should we get mockups for this?

**Comment by jrhizor** [comment]
Actually this is a duplicate of https://github.com/airbytehq/airbyte/issues/8106
